libs/langgraph/tests/test_pregel.py

```diff
@@ -6238,26 +6238,31 @@ def node_3(state: State, store):
         "bar": ["node_1", "subgraph_node_1", "subgraph_node_2", "node_3"],
     }
 
-    assert list(
+    stream_results = list(
         main_graph.stream({"foo": ""}, stream_mode="updates", subgraphs=True)
-    ) == [
-        ((), {"node_1": {"bar": ["node_1"]}}),
-        (
-            (AnyStr("node_2:"),),
-            {"subgraph_node_1": {"foo": "foo", "bar": ["subgraph_node_1"]}},
-        ),
-        (
-            (),
-            {
+    )
+
+    assert len(stream_results) == 5
+    assert stream_results[0] == ((), {"node_1": {"bar": ["node_1"]}})
+    assert stream_results[1] == (
+        (AnyStr("node_2:"),),
+        {"subgraph_node_1": {"foo": "foo", "bar": ["subgraph_node_1"]}},
+    )
+    assert stream_results[2] == (
+        (AnyStr("node_2:"),),
+        {"subgraph_node_2": None},
```

Comment:
shouldn't this be `{"bar": ["subgraph_node_2"]}`?

Reply:
No, that node didn't return any value